cmd/dlv/cmds/commands.go

```diff
@@ -358,6 +358,10 @@ func execute(attachPid int, processArgs []string, conf *config.Config) int {
 
 	var status int
 	if Headless {
+		if Addr == "localhost:0" {
```

Comment:
On second thought, I think it's fine to just print this our irregardless. I don't feel too strongly about this being in log output or normal stdout, but I think it should be printed consistently either way.